File: Easy/123NumberFlip.js
class Solution {
    solve(n) {
        var numStr = n.toString();
        var numArr = numStr.split('');

        for (var i = 0; i < numArr.length; i++) {
            if(numArr[i] !='3'){
                numArr[i] = '3';
                break;
            }
        }

        var numArrJoin = numArr.join('');
        var numStr = parseInt(numArrJoin);

        return numStr;
    }
}

File: Easy/123NumberFlip/123NumberFlip.cpp
int solve(int n)
{
    string s = to_string(n);

    for (int i = 0; i < s.length(); i++)
    {
        if (s[i] != '3')
        {
            s[i] = '3';
            break;
        }
    }

    return stoi(s);
}